test/integration/controller/job/job_controller_test.go

```diff
@@ -46,31 +46,59 @@ import (
 const (
 	parallelism       = 4
 	jobName           = "test-job"
-	jobNamespace      = "default"
 	labelKey          = "cloud.provider.com/instance"
 	priorityClassName = "test-priority-class"
 	priorityValue     = 10
+	parentJobName     = jobName + "-parent"
+	childJobName      = jobName + "-child"
 )
 
 var (
-	wlLookupKey               = types.NamespacedName{Name: workloadjob.GetWorkloadNameForJob(jobName), Namespace: jobNamespace}
 	ignoreConditionTimestamps = cmpopts.IgnoreFields(metav1.Condition{}, "LastTransitionTime")
 )
 
 // +kubebuilder:docs-gen:collapse=Imports
 
-var _ = ginkgo.Describe("Job controller", func() {
+var _ = ginkgo.Describe("Job controller", ginkgo.Ordered, ginkgo.ContinueOnFailure, func() {
 
-	ginkgo.BeforeEach(func() {
+	ginkgo.BeforeAll(func() {
 		fwk = &framework.Framework{
 			ManagerSetup: managerSetup(jobframework.WithManageJobsWithoutQueueName(true)),
 			CRDPath:      crdPath,
 		}
 		ctx, cfg, k8sClient = fwk.Setup()
 	})
-	ginkgo.AfterEach(func() {
+	ginkgo.AfterAll(func() {
 		fwk.Teardown()
 	})
+
+	var (
+		jobNamespace      string
+		ns                *corev1.Namespace
+		wlLookupKey       types.NamespacedName
+		parentWlLookupKey types.NamespacedName
+		childWlLookupKey  types.NamespacedName
+		childLookupKey    types.NamespacedName
```

Comment:
why move all of these variables to this level, if they are only used in one case?